docs/spec/overview.md

```diff
@@ -1,70 +1,93 @@
+- A **Channel** provides event persistance and fanout of events from a
+  well-known input address to multiple outputs described by _Subscriptions_.
 
-# Resource Types
+<!-- This image is sourced from https://drive.google.com/open?id=10mmXzDb8S_4_ZG_hcBr7s4HPISyBqcqeJLTXLwkilRc -->
 
-The primary resources in the Knative Eventing _Feeds_ API are EventSource,
-EventType and Feed:
+![Resource Types Overview](images/resource-types-overview.svg)
 
-* **EventSource**, an archetype of an event producer.
+- **ClusterChannelProvisioners** implement strategies for realizing backing resources
+  for different implementations of _Channels_ currently active in the eventing
+  system.
 
-* **EventType**, the schema for an event. 
+<!-- This image is sourced from https://drive.google.com/open?id=1o_0Xh5VjwpQ7Px08h_Q4qnaOdMjt4yCEPixRFwJQjh8 -->
 
-* **Feed**, the association between the output of an event producer to the
-  input of an event consumer.
+<img alt="Resource Types ClusterChannelProvisioners" src="images/resource-types-provisioner.svg" width="200">
 
-An example _Feeds_ setup: GitHub would be the _EventSource_, a Pull Request
-notification would be the _EventType_ and the _Feed_ describes the _org/repo_,
-credentials, the specific _EventType_ and the consumer of the _Feed_ events.
+Sources are defined by independent CRDs that can be installed into a cluster.
+For more information see [Knative Eventing
+Sources](https://github.com/knative/eventing-sources).
 
+## Subscription
 
-The primary resources in the Knative Eventing _Channels_ API are Channel,
-Subscription and Bus:
+**Subscriptions** describe a flow of events from one _Channel_) to the next
+Channel\* through transformations (such as a Knative Service which processes
+CloudEvents over HTTP). A _Subscription_ controller resolves the addresses of
+transformations (`call`) and destination storage (`result`) through the
+_Targetable_ and _Sinkable_ interface contracts, and writes the resolved
+addresses to the _Channel_ in the `from` reference. _Subscriptions_ do not need
+to specify both a transformation and a storage destination, but at least one
+must be provided.
 
-* **Channel**, a named endpoint which accepts and forwards events.
+All event delivery linkage from a **Subscription** is 1:1 – only a single
+`from`, `call`, and `result` may be provided.
 
-* **Bus**, an implementation of an event delivery mechanism.
+For more details, see [Kind: Subscription](spec.md#kind-subscription).
 
-* **Subscription**, an expressed interest in events from a _Channel_ to be
-  delivered to an HTTP endpoint.
+## Channel
 
-An example _Channels_ setup: A _Channel_ has declared it is implemented by a
-PubSubBusImpl. The PubSubBusImpl _Bus_ accepts input from an event producer
-that is pushing events to the _Channel_. The _Bus_ also watches for
-_Subscription_(s) on the _Channel_ and delivers the events accepted to the
-consumer.
+**Channel** provides an at least once event delivery mechanism which can fan
```

Comment:
We shouldn't be providing any delivery guarantees. That will be dependent on how the logical Channel is backed via its Provisioner. For example, an in-memory Channel implementation that fails between receiving and delivering an event would not conform to an "at least once" guarantee.